core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageWriter.java

```diff
@@ -26,8 +26,8 @@
 import com.linecorp.armeria.internal.common.stream.AbortingSubscriber;
 import com.linecorp.armeria.internal.common.stream.StreamMessageUtil;
 
-abstract class AbstractStreamMessageAndWriter<T> extends CancellableStreamMessage<T>
-        implements StreamMessageAndWriter<T> {
+abstract class AbstractStreamMessageWriter<T> extends CancellableStreamMessage<T>
+        implements StreamMessageWriter<T> {
```

Comment:
Should I make this `abstract class` to `public abstract class` too same with `public interface StreamMessageWrtier`? 🤔

Reply:
I don't think so because it doesn't have any methods or fields that are exposed to public.